pkg/engine/handlers/mutation/mutate_image.go

```diff
@@ -99,7 +100,7 @@ func (h mutateImageHandler) Process(
 				engineapi.RuleError(rule.Name, engineapi.ImageVerify, "failed to fetch secrets", err),
 			)
 		}
-		iv := internal.NewImageVerifier(logger, rclient, h.ivCache, policyContext, *ruleCopy, h.ivm)
+		iv := internal.NewImageVerifier(logger, rclient, h.ivCache, policyContext, *ruleCopy, h.ivm, contextLoader)
```

Comment:
you dont need to add the context loader like this here

Reply:
so can I call it directly from engineapi.EngineContextLoader in imageverifier.go?

Reply:
you are adding variables to the context, why do you have to load the context again, there is no fetching

Reply:
Yes we don't need to load context here.